pkg/codegen/go/gen_program_expressions.go

```diff
@@ -780,6 +797,11 @@ func (g *generator) escapeString(v string) string {
 		if c == '"' || c == '\\' {
 			builder.WriteRune('\\')
 		}
+		if c == '\n' {
```

Comment:
It would be more idiomatic here to use raw string literals (which are escaped with `` ` ``) or to put each string literal on its own line and concatenate them, but what you have is simpler. You might file a follow-up issue to track generating the more idiomatic code.

Reply:
https://github.com/pulumi/pulumi/issues/4855